### PR TITLE
Priority support CH changes

### PR DESCRIPTION
**Pull request by alex-deaconu**
Updated CH version to include ZD integration

Keep extra parameters when refreshing page
Don't reload page when extra parameters are changed only

Close ZD widget when path is updated

Staged at http://1-09-002.rvauser2.appspot.com/

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @fjvallarino please review. Thanks!

File: src/com/risevision/ui/client/common/info/HistoryTokenInfo.java [hunk 1]
```diff
@@ -51,4 +59,20 @@ public String getBookmark() {
...
+
+	public List<String> getExtraTokens() {
+		return extraTokens;
+	}
+
+	public void setExtraTokens(List<String> extraTokens) {
+		this.extraTokens = extraTokens;
+	}
+	
+	public boolean isEqual(HistoryTokenInfo newToken) {
```
Comment (fjvallarino) [inline src/com/risevision/ui/client/common/info/HistoryTokenInfo.java]: @alex-deaconu should we use equals instead of == for String comparison here?

**Comment by alex-deaconu** [comment]
@fjvallarino oops, great catch. Updated by creating a Utils function for String comparison (`null` safe).

Also had to switch staging URLs because App Engine deployments are failing on 002. New URL is http://1-09-003.rvauser2.appspot.com/

Thanks!

**Comment by fjvallarino** [comment]
Changes look good, although I defer to @ezequielc who actually knows rva

Just in case :+1:

**Comment by alex-deaconu** [comment]
@ezequielc @fjvallarino updated to latest CH version. Not sure how I can verify the trial/trial expired cases (need a company id for that). Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Code looks good. Thanks! 👍

**Pull request merged**